source/ZividNETSamples.sln

```diff
@@ -122,6 +128,7 @@ Global
         HideSolutionNode = FALSE
     EndGlobalSection
     GlobalSection(ExtensibilityGlobals) = postSolution
-        SolutionGuid = {FD0FDA0F-31E4-4397-822E-0F1A9A4AED50}
+                SolutionGuid = {FD0FDA0F-31E4-4397-822E-0F1A9A4AED50}
+        SolutionGuid = {90099322-D7FF-42CF-A1D4-4DA13FA774B2}
```

Comment:
Why this change?

Reply:
tab --> 4 spaces

Reply:
reduced indentation to what it was, but now with spaces

Reply:
I don't think adding `SolutionGuid = {90099322-D7FF-42CF-A1D4-4DA13FA774B2}` is needed at all.